util/cmake/SeqAnBuildSystem.cmake

```diff
@@ -426,6 +426,79 @@ macro (seqan_setup_install_vars APP_NAME)
 endmacro (seqan_setup_install_vars)
 
 # ---------------------------------------------------------------------------
+# Macro seqan_install_required_system_libraries ()
+#
+# When packaging apps copy needed dlls
+# ---------------------------------------------------------------------------
+
+macro(INTEL_FILES_FOR_VERSION version)
+  # version can be 2016
+  set(v "${version}")
+
+  # set intel architecture
+  if (CMAKE_MSVC_ARCH STREQUAL "x64")
+    set(CMAKE_INTEL_ARCH "intel64")
+  else ()
+    set(CMAKE_INTEL_ARCH "ia32")
+  endif()
+
+  # Find the runtime library redistribution directory.
+  set(programfilesx86 "ProgramFiles(x86)")
+  find_path(INTEL${v}_REDIST_DIR NAMES ${CMAKE_INTEL_ARCH}/compiler
+    PATHS
+      "$ENV{ProgramFiles}/IntelSWTools/compilers_and_libraries_${v}/windows/redist"
+      "$ENV{${programfilesx86}}/IntelSWTools/compilers_and_libraries_${v}/windows/redist"
+    )
+  mark_as_advanced(INTEL${v}_REDIST_DIR)
+  set(INTEL${v}_FILES_DIR "${INTEL${v}_REDIST_DIR}/${CMAKE_INTEL_ARCH}/compiler")
+
+  if(NOT CMAKE_INSTALL_DEBUG_LIBRARIES_ONLY)
+    set(__install__libs
+      "${INTEL${v}_FILES_DIR}/libmmd.dll"
+      "${INTEL${v}_FILES_DIR}/libiomp5md.dll"
```

Comment:
Can you make installation of this dll depend on `CMAKE_INSTALL_OPENMP_LIBRARIES` ?